package.json

```diff
@@ -88,12 +88,12 @@
   },
   "scripts": {
     "webpack:serve": "webpack-dev-server --config webpack.dev.js",
-    "django:serve": "python manage.py runserver",
+    "django:serve": "python manage.py runserver 0.0.0.0:8000",
     "redis:clear": "redis-cli FLUSHALL",
     "worker:serve": "python manage.py rqworker default short",
     "scheduler:serve": "python manage.py rqscheduler default short",
     "rq:serve": "npm-run-all redis:clear -p worker:serve scheduler:serve",
-    "serve": "run-p django:serve webpack:serve rq:serve",
+    "serve": "run-p django:serve webpack:serve",
```

Comment:
@wlonk Can you say more about this change?

Reply:
I wanted to actually flag it before you reviewed, but you beat me to it. This is just a quick change of convenience, intended to get the Docker image running; the worker stuff was erroring, so I just took it out for now. This is not intended to be the going-forward way.